### DISCUSSION TITLE
Can I set the ws port to be customized when I don’t want it to be on the same port as http?

### DISCUSSION THREAD

**Discussion by rayn316** [erigontech/erigon#8264] [General]
Just like the title, sometimes the same port and geth are different, which leads to increased management costs. I hope you can choose whether to separate the ports.

**Comment by AskAlexSharov** [comment]
"which leads to increased management costs" - how?

**Reply by rayn316** [reply]
In eth, the polygon network will have multiple different clients. The port settings in the configuration file are always the same. But when erigon ws is different from other nodes, different configurations need to be set, and the management software and management configuration need to be updated separately to avoid being unable to service due to port problems.

**Reply by AskAlexSharov** [reply]
we don't have such flag. but probably easy to add. create github issue plz

**Reply by rayn316** [reply]
Yeah, issue has been submitted  https://github.com/ledgerwatch/erigon/issues/8266
